scripts/setup.js

```diff
@@ -40,6 +40,9 @@ document.addEventListener("keypress",function(){
       })
 
       finder.on('end', async () => {
+        state.allFiles.sort(async (a, b)=>{
+          return state.songsMap[a].title.toLowerCase() < state.songsMap[b].title.toLowerCase() ? -1 : 1
+        })
```

Comment:
This much is not enough to show songs list sorted. This sorts `state.allFiles` but you also need to make sure whether the component which is rendering the data is using this or not.

In this case, `songsMap` will be empty and will throw an error for accessing "title" of undefined. For that, ` await extractAndStoreMetaTags()` should be before the sorting mechanism.